var/spack/repos/builtin/packages/costo/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Costo(CMakePackage):
+    """Cosimulation Tools"""
+
+    homepage = "https://gitlab.com/Te_ch/costo"
+    git = "https://gitlab.com/Te_ch/costo.git"
+
+    maintainers("tech-91")
+
+    license("LGPL-3.0-or-later")
+
+    version(
+        "0.0.4", tag="v0.0.4", commit="f9e3ee774a33cc3af566e338d542d7dd0ff36cea", preferred=True
+    )
+    version("develop", branch="devel")
+    version("master", branch="main", deprecated=True)
```

Comment:
There is no `master`: https://gitlab.com/Te_ch/costo/-/branches
There is a `develop`, but the only difference to 0.0.4 is that it's version is 0.0.0. Skip it.
There is a `main` but its only difference is in the CI.

As there is no documentation at all that I could find, I am skeptical what the value of adding it to the main `spack` repo would be.

@tech-91, you you have a good explanation why this is needed as a spack package?

At least a small semblance of some doc what it even is would be required, IMHO.